src/content/es/2020/javascript.md

```diff
@@ -0,0 +1,777 @@
+
+{{ figure_markup(
+  caption="Porcentaje de páginas móviles con al menos una librería de JavaScript vulnerable.",
+  content="83.50%",
+  classes="big-number",
+  sheets_gid="1326928130",
+  sql_file="lighthouse_vulnerabilities.sql"
+) }}
+
+De acuerdo a la auditoría, 83.5% de las páginas móviles usan una librería o _framework_ con al menos una vulnerabilidad de seguridad conocida.
+
+Esto es a lo que llamamos el efecto jQuery. Anteriormente vimos como jQuery es usado en un 83% de las páginas. Múltiples versiones viejas de jQuery contienen vulnerabilidades conocidas, lo cual constituye la vasta mayoría de las vulnerabilidades que esta auditoría revisó.
+
+De las casi 5 millones de páginas móviles que fueron probadas, 81% de ellas contiene una versión vulnerable de jQuery—una delantera considerable contra la segunda librería vulnerable más común—jQuery UI con un 15.6%.
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Librería</th>
+        <th>Páginas vulnerables</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>jQuery</td>
+        <td class="numeric">80.86%</td>
+      </tr>
+      <tr>
+        <td>jQuery UI</td>
+        <td class="numeric">15.61%</td>
+      </tr>
+      <tr>
+        <td>Bootstrap</td>
+        <td class="numeric">13.19%</td>
+      </tr>
+      <tr>
+        <td>Lodash</td>
+        <td class="numeric">4.90%</td>
+      </tr>
+      <tr>
+        <td>Moment.js</td>
+        <td class="numeric">2.61%</td>
+      </tr>
+      <tr>
+        <td>Handlebars</td>
+        <td class="numeric">1.38%</td>
+      </tr>
+      <tr>
+        <td>AngularJS</td>
+        <td class="numeric">1.26%</td>
+      </tr>
+      <tr>
+        <td>Mustache</td>
+        <td class="numeric">0.77%</td>
+      </tr>
+      <tr>
+        <td>Dojo</td>
+        <td class="numeric">0.58%</td>
+      </tr>
+      <tr>
+        <td>jQuery Mobile</td>
+        <td class="numeric">0.53%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>
+    {{ figure_link(
+      caption="Top 10 de librerías que contribuyen la mayor cantidad de páginas móviles vulnerables de acuerdo a Lighthouse.",
+      sheets_gid="1803013938",
+      sql_file="lighthouse_vulnerable_libraries.sql"
+    ) }}
+  </figcaption>
+</figure>
+
+En otras palabras, si podemos hacer que se haga una migración de esas versiones viejas y vulnerables de jQuery, veríamos el número de sitios con vulnerabilidades conocidas desplomarse (al menos hasta que encontremos vulnerabilidades en los _frameworks_ más nuevos).
+
+La mayor parte de las vulnerabilidades encontradas cae en la categoría de severidad "media".
+{{ figure_markup(
```

Comment:
Needs an extra newline or invalid HTML is generated (not sure why HTML validation didn't pick this one up!):

```suggestion

{{ figure_markup(
```